pkg/cryptoauthlib/contrib/atca.c

```diff
@@ -147,7 +147,20 @@ ATCA_STATUS hal_i2c_wake(ATCAIface iface)
     uint8_t data[4] = { 0 };
 
     i2c_acquire(cfg->atcai2c.bus);
+#ifdef MODULE_SAM0_COMMON_PERIPH
```

Comment:
BTW: It would be great if this was a general configuration flag which should then be located (and documented) somewhere in *pkg/cryptoauthlib/include/ataca.h*

Reply:
So you would prefer to let the user decide whether to use this demux/mux wake-up pulse even if this feature is implemented for the used CPU?

(I actually need this feature to ensure the crpyto chip is not woken-up by accident since it's not the only IC attached to the I2C bus ...)

Reply:
I'd rather argue that the feature should be available for other platforms too. It could be a configuration parameter that can be set by a specific platform (board?!) for instance.

Reply:
When #13421 is merged, I would check whether MODULE_PERIPH_I2C_RECONFIGURE is set and if it is, I would use the reconfigure feature. So we don't have to white-list individual boards. This feature would be used on any CPU that supports I2C reconfigure feature.

Or do you see cases where using the reconfigure feature is not desired?

Reply:
> Or do you see cases where using the reconfigure feature is not desired?

@Einhornhool do you?

Reply:
@jue89 have you measured the time it takes to use the demux/mux wake-up pulse?
I implemented a wake pulse using gpios before and it took much longer than sending zeroes (I did it differently, though). I don't know if this demux/mux thing is faster than my gpio wake pulse implementation, but if it's not, then maybe one wouldn't automatically want to use it whenever one uses the reconfigure feature.

Reply:
I'll hook up an oscilloscope and try to measure the required time for the reconfiguration. But it shouldn't be too long from my point of view.

If we are able to drive the crypto device with 400kHz (or even 1MHz) we should save a fair amount of energy even if the reconfiguration takes some additional of time.